modules/dnn/src/cuda/region.cu

```diff
@@ -72,6 +72,80 @@ namespace cv { namespace dnn { namespace cuda4dnn { namespace kernels {
             }
         }
 
+
+
+        template <class T>
+        __global__ void region_box_noact(
+            Span<T> output, View<T> input, View<T> bias,
+            size_type boxes_per_cell, size_type box_size,
+            size_type rows, size_type cols, T scale_x_y,
+            size_type height_norm, size_type width_norm,
+            T object_prob_cutoff)
+        {
+            using vector2_type = get_vector_type_t<T, 2>;
+            auto bias_vPtr = vector2_type::get_pointer(bias.data());
+
+            for (auto box_index : grid_stride_range(output.size() / box_size)) {
+                const auto box_of_the_cell = box_index % boxes_per_cell; /* box number within a cell */
+                const auto box_offset = box_index * box_size;
+
+                const auto batch_inner_size = rows * cols * boxes_per_cell;
+                const auto row_inner_size = cols * boxes_per_cell;
+                const auto col_inner_size = boxes_per_cell;
+
+                const auto y = (box_index % batch_inner_size) / row_inner_size;
+                const auto x = (box_index % row_inner_size) / col_inner_size;
+
+                const auto tmp_x = ((input[box_offset + 0]) - static_cast<T>(0.5)) * scale_x_y + static_cast<T>(0.5);
```

Comment:
I think duplicating the whole kernel for changing two lines is overkill. Can you please add an argument instead?

Reply:
Modified code to use a parameter in the function.

Reply:
@YashasSamaga is right. Could you, please split the patch on 2 parts:
- CPU version for 3.4. We merge 3.4 to master weekly or biweekly and the solution appears in master too.
- CUDA version for master only. When the first patch is ready.

Reply:
I propose to not close the PR, but create new one for 3.4 first.